simple-search/service.py

```diff
@@ -0,0 +1,44 @@
+from authority import API_KEY
+from json import dumps, loads
+from logging import error
+from requests import get
+
+
+class SearchService:
+    SEARCH = 'https://openapi.companybooknetworking.com/1.0/search/search?'
```

Comment:
I was thinking about that when I made the change O2 suggested. In principle I agree. Then there are different search service endpoints and this particular one is actually called search. If I used the match endpoint, would it be called ENDPOINT?

More importantly, if we supported both match and search, what would it be called?